.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
     },
     sourceType: "module"
   },
+  ignorePatterns: ['node_modules'],
```

Comment:
Added this because eslint was tripping up on the typescript files inside `src/twilio/node_modules`

Reply:
i ran into this too, i ended up adding a .eslintignore file - this way is probably more elegant!